src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -89,25 +91,23 @@ export const Details: FC<DetailsProps> = ({
     conversation.items?.[0]?.item?.__typename !== "%other" &&
     conversation.items?.[0]?.item
 
-  // @ts-expect-error STRICT_NULL_CHECK
-  const attachments = conversation.messagesConnection.edges
-    // @ts-expect-error STRICT_NULL_CHECK
-    .map(({ node }) => node.attachments)
-    .filter(attachments => attachments.length > 0)
-    .reduce((previous, current) => previous.concat(current), [])
-    .filter(attachment => !attachment.contentType.includes("image"))
+  const attachments = conversation?.messagesConnection?.edges
+    ?.map(edge => edge?.node?.attachments)
+    ?.filter(attachments => attachments?.length)
+    ?.reduce((previous, current) => previous?.concat(current!), [])
+    ?.filter(attachment => !attachment?.contentType.includes("image"))
 
   const attachmentItems = attachments?.map(attachment => {
     return (
       <Link
-        key={attachment.id}
-        href={attachment.downloadURL}
+        key={attachment?.id ?? undefined}
+        href={attachment?.downloadURL ?? undefined}
```

Comment:
Rather than creating undefined links, might we guard against the existence of `downloadUrl` and then conditionally render? That or filter `attachments` so only truthy values appear.